src/frontend/src/components/NotificationManager/components/Notification/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Notification as SDSNotification } from "czifui";
+import { FC, ReactNode } from "react";
+import { deleteNotification } from "src/common/redux/actions";
+import { useDispatch } from "src/common/redux/hooks";
+import { ReduxNotification } from "src/common/redux/types";
+import { CreateNSTreeFailureNotif } from "../CreateNSTreeFailureNotif";
+import { CreateNSTreeSuccessNotif } from "../CreateNSTreeSuccessNotif";
+import { DownloadFilesFailureNotif } from "../DownloadFilesFailureNotif";
+import { SendInviteFailureNotif } from "../SendInviteFailureNotif";
+import { SendInviteSuccessNotif } from "../SendInviteSuccessNotif";
+import { UsherPlacementSuccessNotif } from "../UsherPlacementSuccessNotif";
+
+/**
+ * We should only store things that are directly serializable (strings, numbers, etc) in redux.
```

Comment:
I agree with you so, so much. I have seen components and functions go into redux and it just feels so wrong